### ISSUE TITLE
Including files outside the User folder

### ISSUE THREAD

**Issue by Carlovan** [mfuentesg/SyncSettings#71]
I have a lot of snippets in `Packages/Cpp/Snippets` folder and I'd love to be able to sync them. 
I tried including `"../Cpp/Snippets"` in the `included_files` configuration option but it doesn't work.

**Comment by Trevor-** [comment]
Yep, I'd like this feature too

**Comment by itslooklike** [comment]
Hi! absolute path didn`t work too?
I try
```
{
	"included_files":
	[
		"~/emmet/preferences.json"
	]
}
```
but `emmet` not add in gist

**Comment by utjduo** [comment]
I'm trying to get this to work but what ever I do it doesn't upload.
My folder is "C:\Users\Musikdator\AppData\Roaming\Sublime Text 3\Packages\CC-Extension-Builder-for-Sublime-Text"
I have tried with both \\ and / slashes. I have tried single files also but it just won't upload.

This is how my code looks like:
```  "included_files": [
    "C:/Users/Musikdator/AppData/Roaming/Sublime Text 3/Packages/yay.test",
  ]```
(as said, I have tried both / and \\) Have I done something wrong?
